src/api/videoRoom.ts

```diff
@@ -0,0 +1,79 @@
+import Bugsnag from "@bugsnag/js";
+import firebase from "firebase/app";
+
+import { VideoChatRequest, VideoChatRequestState } from "types/VideoRoom";
+
+export const inviteToVideoChat = async (
+  hostUserId: string,
+  hostUserLocation: string,
+  invitedUserId: string
+) => {
+  const videoRoomRequest: VideoChatRequest = {
+    hostUserId: hostUserId,
+    hostUserLocation: hostUserLocation,
+    invitedUserId: invitedUserId,
+    invitedUserLocation: "",
+    state: VideoChatRequestState.Invited,
+    createdAt: firebase.firestore.FieldValue.serverTimestamp(),
+  };
+
+  return await firebase
+    .functions()
+    .httpsCallable("videoRoom-addVideoRoomRequest")({
```

Comment:
Instead of `addVideoRoomRequest` this would probably be better as `inviteToVideoChat` or similar. Name it for it's 'real world function', not it's underlying implementation details.